solutions/CF Jury Marks.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: jury-marks
+source: CF
+title: Jury Marks
+author: Danh Ta Chi Thanh
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/53302)
+
+**Time Complexity:** $\mathcal{O}(K^2 \log K)$
+
+<LanguageSection>
+
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);
+	int K, N;
+	cin >> K >> N;
+	vector<int> a, b;

```

Comment:
write `vector<int> a(K+1), b(N);` and remove the resizes

Reply:
sorry, it's my bad habit :/